packages/app/dev.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_RECAPTCHA_SITE_KEY=fake-site-key
```

Comment:
same with this?

Reply:
resolving this, maybe to be added to like an INSTALLING.md (or whatever we decide to name it) later